fastapi_restful/cbv.py

```diff
@@ -85,6 +85,15 @@ def new_init(self: Any, *args: Any, **kwargs: Any) -> None:
     setattr(cls, CBV_CLASS_KEY, True)
 
 
+def _remove_router_tags(route: Route, router: APIRouter) -> None:
+    """
+    Removes tags previously assigned to the route by the router, as they will
+    be added again later when re-added to the router.
+    """

```

Comment:
IMO no need to comment this code since the function name really tells us whats going on but I'm not picky